portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro

```diff
@@ -93,6 +93,8 @@ definition {
 
 		Navigator.openSpecificURL(url = "${analyticsCloudURL}/admin");
 
+		WaitForPageLoad();
```

Comment:
Hey @joshitagaki, Can you put `ACUtils.waitForLoading();` after this one too? I think it would be good to check that the loading icon is not on the screen before the condition to check if you need to change the password is executed